ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -302,124 +371,267 @@ export const Homepage = withUserProfile()(class extends React.Component<
-                                        twoFactorAuthCompleted={twoFactorAuthCompleted}
-                                        dataUseAgreementCompleted={dataUseAgreementCompleted}/>
-                ) : (
-                  <div style={{display: 'flex', flexDirection: 'row', paddingTop: '2rem'}}>
-                    <div style={styles.contentWrapperLeft}>
-                      <div style={styles.mainHeader}>Researcher Workbench</div>
-                      <TooltipTrigger content={<div>Your Firecloud billing project is still being
-                        initialized. Workspace creation will be available in a few minutes.</div>}
-                                      disabled={canCreateWorkspaces}>
-                        <CardButton disabled={!canCreateWorkspaces}
-                                    onClick={() => navigate(['workspaces/build'])}
-                                    style={{margin: '1.9rem 106px 0 3%'}}>
-                          Create a <br/> New Workspace
-                          <ClrIcon shape='plus-circle' style={{height: '32px', width: '32px'}}/>
-                        </CardButton>
-                      </TooltipTrigger>
-                    </div>
-                    <div style={styles.contentWrapperRight}>
-                      <a onClick={() => navigate(['workspaces'])}
-                         style={{fontSize: '14px', color: colors.white}}>
-                        See All Workspaces</a>
-                      <div style={{marginRight: '3%', display: 'flex', flexDirection: 'column'}}>
-                        <div style={{color: colors.white, height: '1.9rem'}}>
-                          <div style={{marginTop: '.5rem'}}>Your Last Accessed Items</div>
-                        </div>
-                        <RecentWork dark={true}/>
-                      </div>
-                    </div>
-                  </div>)
-                ) :
-              <Spinner dark={true} style={{width: '100%', marginTop: '5rem'}}/>}
-          </div>
-        </div>
-        <div>
-          <div style={styles.quickRow}>
-            <div style={styles.quickTourLabel}>Quick Tour & Videos</div>
-            {quickTourResources.map((thumbnail, i) => {
-              return <React.Fragment key={i}>
-                <Clickable onClick={thumbnail.onClick}
-                           data-test-id={'quick-tour-resource-' + i}>
-                  <img style={{maxHeight: '121px', width: '8rem', marginRight: '1rem'}}
-                       src={thumbnail.src}/>
-                </Clickable>
-              </React.Fragment>;
-            })}
+    if (environment.enableHomepageRestyle) {
+      return <React.Fragment>
+        <div style={styles.pageWrapper}>
+          <div style={styles.welcomeMessage}>
+            <div style={{display: 'flex', flexDirection: 'column', width: '50%'}}>
+              <div style={{display: 'flex', flexDirection: 'row'}}>
```

Comment:
Since display: 'flex' and flexDirection is being used in so many style i wonder if we can extract it and use it in other styles as well as here

Reply:
I extracted it out so that we can see what it would look like. I didn't change it for the code that will be deleted.

Reply:
https://github.com/all-of-us/workbench/pull/2511/commits/05dbb8b3bc9c3e0063efc94b89bab19ee31f0e87

Reply:
thoughts?